src/Lean/PrettyPrinter/Delaborator/Basic.lean

```diff
@@ -106,7 +106,16 @@ register_builtin_option pp.proofs.withType : Bool := {
   group    := "pp"
   descr    := "(pretty printer) when eliding a proof (see `pp.proofs`), show its type instead"
 }
-
+register_builtin_option pp.pi_motives : Bool := {
```

Comment:
We switched to camelCase option names at some point, though we haven't updated all old names yet

Reply:
Fixed. Using these three now:
```lean
set_option pp.motives.pi true
set_option pp.motives.nonConst true
set_option pp.motives.all true
```